tweb/src/main/resources/templates/pages/requests/detail.html

```diff
@@ -108,43 +106,59 @@ <h1 class="mb-2 media">
                         </div>
                     </div>
                 </div>
-
-                <div class="card comment" v-if="phaseFilter === 'comments'" th:each="comment : ${githubComments}">
-                    <div class="card-body">
-                        <div class="request-details d-flex flex-wrap flex-md-nowrap align-items-start">
-                            <div class="request-details__logo">
-                                <div>
-                                    <img th:src="@{${comment.userAvatar}(size=45)}" th:alt-title="${comment.userName}"/>
-                                </div>
-                                <div class="request-details__owner mb-3">
-									<font-size-fit v-bind:max-size="15" v-bind:key="'username'" th:text="${comment.userName}">FundRequest</font-size-fit>
-                                </div>
-                                <div class="request-details__last-modified-at">
-                                    <small>Commented</small>
-                                    <font-size-fit v-bind:max-size="15">
-                                        <timeago th:attr="'v-bind:since'='&quot;' + ${comment.createdAt} + '&quot;'" v-bind:auto-update="60"></timeago>
-                                    </font-size-fit>
+                <div class="row mb-5" v-if="phaseFilter === 'info'">
+                <div class="col-12">
+                    <h2 class="section-title">Comments</h2>
+                    <h3 class="section-subtitle">
+                        Comments posted on the request
+                    </h3>
+                </div>
+                <div class="col-12">
+                    <div class="card comment">
+                        <div class="card-body">
+                            <div th:each="comment : ${githubComments}">
```

Comment:
the th:each should be on the ```<div class="card comment">```. (See difference on you local machine, and production)

Reply:
checking it :)

Reply:
fixed Steve, sorry for it